docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -547,9 +570,11 @@ spec:
     inputRefs:
     - application
     - infrastructure
-    - audit
 ----
 ====
+<1> `serviceAccount.name` must have permissions to collect logs AND send to loki gateway
```

Comment:
do we mention how to create the write permissions?  I wonder if we should add a validation check to notify users that its missing

Reply:
I'll add it.
A validation would help the customer.  It may otherwise not be obvious.  I'll take a look.